manifests/transport.pp

```diff
@@ -49,7 +49,12 @@
 ) {
   include ::postfix::augeas
 
-  $smtp_nexthop = (String($nexthop) =~ /\[.*\]/)
+  if ($nexthop) {
+    $smtp_nexthop = ($nexthop =~ /\[.*\]/)
+  }
+  else {
+    $smtp_nexthop = undef
+  }
```

Comment:
This block could be achieved with a selector.

Reply:
What is the need to change this actually?

Reply:
Reset till the start of initial update changes and rebased from `camptocamp:master #291`